manim/utils/module_ops.py

```diff
@@ -115,6 +115,9 @@ def prompt_user_for_choice(scene_classes):
         sys.exit(2)
     except EOFError:
         sys.exit(1)
+    except ValueError:
+        print("No scenes chosen. Exiting")
```

Comment:
This should use our logging system instead of a print statement
```suggestion
        logger.error("No scenes were selected. Exiting.")
```